### PR TITLE
Update OpenResty to 1.17.8.2

### PR DESCRIPTION
**Pull request by seanhamlin**
# Checklist
- [x] Affected Issues have been mentioned in the Closing issues section
- [x] Documentation has been written/updated
- [x] PR title is ready for changelog and subsystem label(s) applied

Explain the **details** for making this change. What existing problem does the pull request solve?

Updates OpenResty (Nginx) to the most recent version, based on NGINX 1.17.8 core (https://openresty.org/en/changelog-1017008.html), still running on Alpine 3.11

### CODE REVIEW

File: images/nginx/Dockerfile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ARG IMAGE_REPO
 FROM ${IMAGE_REPO:-lagoon}/commons as commons
-# alpine 3.11 per https://github.com/openresty/docker-openresty/blob/master/alpine/Dockerfile
-FROM openresty/openresty:1.15.8.2-alpine
+# Alpine 3.11 per https://github.com/openresty/docker-openresty/blob/master/alpine/Dockerfile#L5
+FROM openresty/openresty:1.17.8.2-1-alpine
```
Comment (tobybellwood) [inline images/nginx/Dockerfile]:
```suggestion
FROM openresty/openresty:1.17.8.2-alpine
```
Suggest to pin to the minor release tag to ensure we get any patches that come through

Comment (seanhamlin) [reply images/nginx/Dockerfile]: Good call, I force pushed a new commit.

Comment (tobybellwood) [reply images/nginx/Dockerfile]: To a tag that doesn't exist yet :facepalm:

Comment (seanhamlin) [reply images/nginx/Dockerfile]: doh. the `.3` tag exists in OpenResty, just no in DockerHub. Perhaps `openresty/openresty:1.17.8.2-alpine` ?

**Review by smlx — APPROVED** [review]
LGTM

**Pull request merged**